src/main/java/network/brightspots/rcv/ContestConfig.java

```diff
@@ -88,8 +89,12 @@ class ContestConfig {
   private final String sourceDirectory;
   // Used to track a sequential multi-seat race
   private final List<String> sequentialWinners = new LinkedList<>();
-  // Mapping from candidate code to full name
-  private Map<String, String> candidateCodeToNameMap;
+  // Canonical candidate names (no aliases or codes)
+  private Set<String> candidateNames;
+  // Mapping from any candidate alias to the candidate's display name
+  private Map<String, String> candidateAliasesToNameMap;
+  // Mapping from any candidate alias to the candidate's code
+  private Map<String, String> candidateAliasesToCodeMap;
```

Comment:
Note: this is only used when outputting the result CSVs

Reply:
I think we should delete this and all related code after confirming with @rrojas350, per my other comment.

Reply:
Sounds good - shall we do that in a separate PR to show that this code has no side-effects? Or do you prefer it all together?

Reply:
A follow-up PR sounds good, since this one is already a monster. Would you mind filing an issue for this or noting it in an existing related issue that we make sure to handle before closing? You could link to this comment thread for context.

Reply:
https://github.com/BrightSpots/rcv/issues/663